800.renames-and-merges/g.yaml

```diff
@@ -512,7 +512,7 @@
 - { setname: gtkgreet,                 name: greetd-gtkgreet }
 - { setname: gtkhotkey,                name: libgtkhotkey }
 - { setname: gtkhtml,                  namepat: "(?:lib|compat-)?gtkhtml[0-9.-]*" }
-- { setname: gtkmm,                    namepat: "(?:lib)?gtkmm[0-9.+-]*(-docs|-documentation|-reference)?", addflavor: $1 }
+- { setname: gtkmm,                    namepat: "(?:lib)?gtkmm(?:(-docs|-documentation|-reference)[0-9.+-]*|[0-9.+-]*(-docs|-documentation|-reference)?)", addflavor: "$1$2" }
```

Comment:
Um, what is this supposed to do? If you want to include `gtkmm-documentation3.0`, you only need to append `[0-9.-]?` to the pattern.

Reply:
Sorry, `*`, not `?`